### PR TITLE
fix method name updateLineItems() for shopify-buy >= v1

### PR DESCRIPTION
**Pull request by do7be**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/47742, https://github.com/Shopify/js-buy-sdk#updating-line-items, https://github.com/Shopify/js-buy-sdk/blob/master/tutorials/MIGRATION_GUIDE.md#updating-line-items
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

I don't know all types are correct for latest version. So I didn't update version.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## shopify-buy ([<kbd>unpkg</kbd>](https://unpkg.com/browse/shopify-buy@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'shopify-buy'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 768d3e232544967bd29fd4ebb2e8eb0a0e5de109
</p>

File: types/shopify-buy/index.d.ts [hunk 1]
```diff
@@ -95,7 +95,7 @@ declare namespace ShopifyBuy {
         /**
          * Update a line item quantity based on line item id
          */
-        updateLineItem(checkoutId: string | number, lineItems: AttributeInput[]): Promise<Cart>;
```
Comment (davidthrasher) [inline types/shopify-buy/index.d.ts:98]: image

My original error above. Are we sure this didn't get updated already? It looks like the name is being changed to `updateLineItems` plural again. Even though that was the original issue.

**Review by straiforos — APPROVED** [review]
Sorry for the delay. Seems reasonable enough! Thank you for your contributions.

**Comment by do7be** [comment]
Ready to merge

**Pull request merged**